auth-tokens-filter/src/main/java/com/palantir/tokens/auth/http/BearerTokenLoggingFeature.java

```diff
@@ -67,14 +68,14 @@ public void configure(ResourceInfo resourceInfo, FeatureContext context) {
             throw new SafeIllegalStateException(
                     "Multiple parameters annotated with @HeaderParam('Authorization')",
                     SafeArg.of("class", resourceInfo.getResourceClass()),
-                    SafeArg.of("method", resourceInfo.getResourceMethod()));
+                    SafeArg.of("method", Objects.toString(resourceInfo.getResourceMethod())));
         }
 
         if (authorizationHeaderParams.isPresent() && authorizationHeaderParams.get().size() == 1) {
             log.debug(
                     "Enabling BearerTokenLoggingFilter",
                     SafeArg.of("class", resourceInfo.getResourceClass()),
-                    SafeArg.of("method", resourceInfo.getResourceMethod()));
+                    SafeArg.of("method", Objects.toString(resourceInfo.getResourceMethod())));
```

Comment:
toString is more expensive than anything else here and can be avoided unless debug is enabled

Reply:
This only happens once, when you register a resource, so I'd actually be in favor of lowering this to info.

Reply:
👍

Reply:
Would be nice to have a more general pattern than SafeArg.of("name", Objects.toString(value)), we've repeated that a lot.

Reply:
Happy to decouple that from this PR though.

Reply:
Think more work has to be done in automating doing the right thing for sure, but I think it'll probably be easier to have different logic for different arg types (in the form of a chain of runtime type matchers) so that service or library authors don't have to think about whether their arg should be toString'ed or not.